source

```diff
@@ -102464,13 +102467,15 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    character's code point) to the current attribute's name.</dd>
 
    <dt>U+0000 NULL</dt>
-   <dd><span>Parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current
+   <dd>This is an <span data-x="parse-error-unexpected-null-character">unexpected-null-character</span>
+   <span>parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current
    attribute's name.</dd>
 
    <dt>U+0022 QUOTATION MARK (&quot;)</dt>
    <dt>U+0027 APOSTROPHE (')</dt>
    <dt>U+003C LESS-THAN SIGN (&lt;)</dt>
-   <dd><span>Parse error</span>. Treat it as per the "anything else" entry below.</dd>
+   <dd>This is an <dfn data-x="parse-error-invalid-character-in-attribute-name">invalid-character-in-attribute-name</dfn>
```

Comment:
Regarding this particular error let's name it `unexpected-character-in-attribute-name`